File: follow-the-beat/src/main/java/com/maco/followthebeat/v2/core/dto/StageDTO.java
package com.maco.followthebeat.v2.core.dto;

import lombok.*;

import java.util.List;
import java.util.UUID;

@EqualsAndHashCode(callSuper = true)
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class StageDTO extends LocationDTO {
    private UUID id;
    private String name;
    private FestivalDTO festivalDTO;
    private List<ConcertDTO> dtoConcerts;
}

File: follow-the-beat/react-beats/src/types/StageDto.ts
import { LocationDto } from "./LocationDto";
import {ConcertResp} from "./ConcertResponseDto.ts";
import {FestivalDto} from "./FestivalDto";

export interface StageDto extends LocationDto {
    id: string;
    name: string;
    festivalDTO: FestivalDto;
    dtoConcerts: ConcertResp[];
}
